src/App.tsx

```diff
@@ -62,6 +62,13 @@ const App = () => {
       <AuthenticatedRoute path="/scan" exact>
         <ScanPage />
       </AuthenticatedRoute>
+      <AuthenticatedRoute
+        path="/add-test"
+        exact
+        requiredPermissions={['CREATE_USERS']} // TODO: Add 'CREATE_TESTS_WITHOUT_ACCESS_PASS' when test adding is there
```

Comment:
It's merged now

Reply:
Yup, at the moment this PR is just not adding the tests yet, so not really using this permission 👍